plyer/platforms/linux/screenshot.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+from plyer.facades import Screenshot
+from plyer.utils import whereis_exe
+
+
+class XwdScreenshot(Screenshot):
+    def __init__(self, file_path=None):
+        default_path = 'test.jpg'
+        super(XwdScreenshot, self).__init__(file_path or default_path)
+
+    def _take_shot(self):
+        subprocess.call(["xwd", "-root", "-out", "screenshot.xwd"])
+        subprocess.call(["convert", "screenshot.xwd", self.file_path])
+        subprocess.call(["rm", "screenshot.xwd"])
+
+
+def instance():
+    return XwdScreenshot()

```

Comment:
Use the imported `whereis_exe` to test if `xwd` is available. In this shape it'll return Xwd even on Wayland, where such thing most likely isn't available (Wayland != Xserver).

Also, don't forget to test if `convert` is available too, or it'll happily throw an error.